### PR TITLE
Added more explicit examples on setting custom metadata from gsutil

### PR DESCRIPTION
**Pull request by estebanbouza**

### CODE REVIEW

File: gslib/addlhelp/metadata.py [hunk 1]
```diff
@@ -153,8 +153,11 @@
 
     gsutil -h x-goog-meta-reviewer:jane cp mycode.java gs://bucket/reviews
 
-  You can add multiple differently-named custom metadata fields to each object.
+  would store the custom metadata key-value pair of "reviewer" and "jane". You can add multiple differently-named custom metadata fields to each object. For example:
```
Comment (thomasmaclean) [inline gslib/addlhelp/metadata.py]:
```
  You can also add multiple differently-named custom metadata fields to each
  object. For example, to add the additional key-value pair `approver` and
  `john`:
```
The text for these lines should break at 80 characters.

File: gslib/addlhelp/metadata.py [hunk 2]
```diff
@@ -153,8 +153,11 @@
```
Comment (thomasmaclean) [inline gslib/addlhelp/metadata.py]: Move your big about what they key-value pairs would be up here:
```
  to a Google Cloud Storage object by using "x-goog-meta" with -h. For example,
  to add the key-value pair `reviewer` and `jane`:
```

File: gslib/addlhelp/metadata.py [hunk 3]
```diff
@@ -153,8 +153,11 @@
 
     gsutil -h x-goog-meta-reviewer:jane cp mycode.java gs://bucket/reviews
 
-  You can add multiple differently-named custom metadata fields to each object.
+  would store the custom metadata key-value pair of "reviewer" and "jane". You can add multiple differently-named custom metadata fields to each object. For example:
 
+    gsutil -h x-goog-meta-reviewer:jane -h x-goog-meta-approver:padin cp mycode.java gs://bucket/reviews
+
+  would store two custom metadata key-value pairs of "reviewer": "jane" and "approver": "padin".
```
Comment (thomasmaclean) [inline gslib/addlhelp/metadata.py]:
```
    gsutil -h x-goog-meta-reviewer:jane -h x-goog-meta-approver:padin \\
           cp mycode.java gs://bucket/reviews
```
Again, splitting lines at 80 characters.

**Comment by NickGoog** [comment]
Closing b/c page has been edited since this PR: https://github.com/GoogleCloudPlatform/gsutil/pull/1173

**Pull request closed without merging**